src/model.rs

```diff
@@ -1588,6 +1597,37 @@ fn decode_discriminator(value: Value) -> Result<String> {
 	}
 }
 
+fn decode_notes(value: Value) -> Result<BTreeMap<UserId, String>> {
+	let notes = match value.as_object() {
+		Some(object) => {
+			let mut notes: BTreeMap<UserId, String> = BTreeMap::new();
+
+			for (key, value) in object.iter() {
```

Comment:
If you use `.into_iter()` here, you can probably avoid the String copy at `String::from` later in this method. Use `into_string` if needed.